src/pages/NotFound.test.js

```diff
@@ -0,0 +1,37 @@
+// Imports React into our test file.
+import React from 'react'
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import NotFound from './NotFound.js'
+import questionbox from '../img/questionbox.png'
+
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({ adapter: new Adapter() })
+
+
+describe("When NotFound renders", () => {
+    let notFound
+    beforeEach(() => {
+        notFound = shallow(<NotFound />)
+    })
+
+    it("displays a heading ", () => {
+        const heading = notFound.find('p')
```

Comment:
Heading is a style of tag that is presented with h1-h6. Paragraph is a tag that is represented with a "p." Please label your variables accordingly.